File: app/src/main/java/com/work/lazxy/writeaway/event/EventCompressComplete.java
package com.work.lazxy.writeaway.event;

/**
 * Created by Lazxy on 2017/5/30.
 */

public class EventCompressComplete {
    public boolean mIsSuccessful;
    public String mErrorMessage;
    public int mCurrentIndex;
    public int mTaskSum;

    public EventCompressComplete(int index, int taskSum,boolean isSuccessful) {
        mIsSuccessful = isSuccessful;
        mCurrentIndex = index;
        mTaskSum = taskSum;
    }

    public EventCompressComplete(boolean isSuccessful,String errorMessage){
        mIsSuccessful = isSuccessful;
        mErrorMessage = errorMessage;
    }
}

File: app/src/main/java/com/work/lazxy/writeaway/event/EventCompressComplete.kt
package com.work.lazxy.writeaway.event

/**
 * Created by Lazxy on 2017/5/30.
 */
class EventCompressComplete {
    @JvmField
    var mIsSuccessful: Boolean

    @JvmField
    var mErrorMessage: String? = null

    @JvmField
    var mCurrentIndex = 0

    @JvmField
    var mTaskSum = 0

    constructor(index: Int, taskSum: Int, isSuccessful: Boolean) {
        mIsSuccessful = isSuccessful
        mCurrentIndex = index
        mTaskSum = taskSum
    }

    constructor(isSuccessful: Boolean, errorMessage: String?) {
        mIsSuccessful = isSuccessful
        mErrorMessage = errorMessage
    }
}